example/test.py

```diff
@@ -159,23 +159,19 @@ def test_database_tables_created(self):
         count = int(psql_output.strip())
         assert_that(count, GreaterThan(0))
 
-    def get(self, path, **kwargs):
-        return requests.get(
-            'http://127.0.0.1:{}{}'.format(self.web_port, path), **kwargs)
-
-    def test_admin_site_live(self):
+    def test_admin_site_live(self, web_client):
         """
         When we get the /admin/ path, we should receive some HTML for the
         Django admin interface.
         """
-        response = self.get('/admin/')
+        response = web_client('/admin/')
```

Comment:
Afaics, requests doesn't have a way to build URLs with a domain + path, it just takes the whole domain. And I can't think of a neat way of doing so.